src/main/java/core/basesyntax/storage/Storage.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.storage;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
```

Comment:
you can make additional DAO layer which contains all db manipulations and leave this class as db mock